### PR TITLE
Added the student response CSV file and the PDF file

### LINKED ISSUE #92: Create Dataset for testing: Ground Truth
We have to create Ground truth dataset for custom sheets, so we can test the performance of custom sheets with this help of this. This requires to create the true answers for the scans that was sent by Dr. Holdener in project channel (for testing ai results).
This issue would need to manually go through each sheet and generate a CSV version of the answers. You will need to set up way to compare the true answers with AI produced answers.
set up a script that takes 2 Input CSV, 1 for true answer that you created and other will be output from bubble scan, compares against the true answers.

Success Criteria:
- [x] Set up the comparison scripts.
- [x] Get the output in digital format.
- [ ] Then scan those with our application for accuracy.
- [ ] Then compare the output with your comparison scripts.
- [ ] Report the results and misses.

Data:
[bubble_scans_50_5.pdf](https://github.com/user-attachments/files/17479271/bubble_scans_50_5.pdf)

### PR DESCRIPTION
**Pull request by yildizahmetali**
Fixes #92 

**What was changed?**
Created a new folder named StudentResponses and added two files: studentResponses pdf file and studentAnswers csv file. Which can be used to comparation.

*Here, describe what part of the application you changed (e.g. login page, database, etc.). If possible, mention what specific components were added, removed, or modified.*

The folder I created is under the InputData folder.
[bubble_scans_50_5.pdf](https://github.com/user-attachments/files/17547189/bubble_scans_50_5.pdf)
[bubblescan_answers_50.csv](https://github.com/user-attachments/files/17547190/bubblescan_answers_50.csv)

**Why was it changed?**

We needed these data to test whether our algorithm works efficiently and correctly.

### CODE REVIEW

**Review by ramezmosad — COMMENTED** [review]
I checked out your branch and ran the program, and that didn't cause any errors. I also see the StudentResponses PDF and StudentAnswers CSV file under the InputData directory and they both look good. I believe everything looks good! Great job!!

**Comment by kate-holdener** [comment]
@yildizahmetali You need to stop committing files that should not be committed. Look at the "Files changed" tab of your PR!

**Comment by kate-holdener** [comment]
@yildizahmetali You are missing the script for comparing the expected with actual

File: .DS_Store
Comment (Chirag2x) [inline .DS_Store:1]: @yildizahmetali, Please try to reactor your code and be informative wha you are adding. The same file was also added last time by team mates then we had to refactor again. So be careful, we may generate bunch of garbled files into the main.

**Review by Chirag2x — CHANGES_REQUESTED** [review]
Why did I am seeing more than 800 files for review. Refactor this and add only things which is important, I believe it should be Original PDF, Respective CSV files, and the script for the comparison, For the script you know the format of output you are gonna get from bubble Scan and you have your csv. So this would be easy task to develop. We are gonna use it in near future.

**Comment by Chirag2x** [comment]
Due to the additional file we are having a merge conflicts.

File: .DS_Store
Comment (Chirag2x) [reply .DS_Store:1]: Yes, Good Job.

**Pull request closed without merging**